source/main.cpp

```diff
@@ -129,7 +129,7 @@ int main(int argc, char **argv) {
 	int patchfile = 0;
 	int header;
 	rawDataOffset=0;
-	char datName[]="YS:/" DATNAME;
+	char datName[MAX_DATNAME_LEN]="YS:/" DATNAME;
```

Comment:
@Syphurith I don't need to do that, coz I think this is the smallest changes to original codes to fix the problem.